### DISCUSSION TITLE
Figures

### DISCUSSION THREAD

**Discussion by Pixflowave** [EcrituresNumeriques/stylo#1138] [Q&A]
Bonjour,
Je cherche à écrire un article avec figures / Notes de bas de pages. Les notes de bas de pages automatisées fonctionnent bien mais l'import d'image ne semble pas opérationnel ... avez-vous une médiathèque ? 

la syntaxe markdown suivante est-elle juste ?
```
[Voir la Figure 1](#figure1)
```

Et définir la figure avec un identifiant : 

```
![Figure 1 : Description](chemin/image.png "Légende de l'image"){#figure1}
```

**Comment by thom4parisot** [comment]
Ça sera à suivre avec #1392 et #1591 ; comme implémentations de stockage de fichiers gérés par les utilisateur·ices.

Pour l'instant y'a cette fonctionnalité cachée #1465, qui stocke sur un serveur pas forcément très fiable/pérenne.
